pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -935,6 +935,14 @@ func (s *DiscoveryServer) removeCon(conID string, con *XdsConnection) {
 	}
 }
 
+func (s *DiscoveryServer) workloadLabels(proxy *model.Proxy) model.Labels {
+	var workloadLabels model.Labels
+	if workload := s.WorkloadsByID[proxy.IPAddresses[0]]; workload != nil {
```

Comment:
I think getting `IPAdresses[0]` is safe but not 100% sure if someone could verify

Reply:
That's a good point.
I found code that also simply accesses `Proxy.IPAddresses[0]`, but now found that this in the kube registry, where it makes sense to make this assumption.
I am going to change this, even if this removes the precious lgtm label 😢